checkov/terraform/checks/resource/aws/KeyspacesTableUsesCMK.py

```diff
@@ -0,0 +1,25 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class KeyspacesTableUsesCMK(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure Keyspaces Table uses CMK"
+        id = "CKV_AWS_265"
+        supported_resources = ['aws_keyspaces_table']
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf) -> CheckResult:
```

Comment:
why not just use a `BaseResourceValueCheck` with an inspected key of `encryption_specification/[0]/type`? or is it possible to set it without actually setting `kms_key_identifier`?

Reply:
someone code have written the tf that way, id have preferred what you say - and it was originally that way.

Reply:
just went for the safer option

Reply:
ok gotcha